src/sentry/grouping/strategies/newstyle.py

```diff
@@ -278,7 +294,10 @@ def get_function_component(
     ids=["frame:v1"],
     interface=Frame,
 )
-def frame(frame, event, context, **meta):
+def frame(
+    interface: Frame, event: Event, context: GroupingContext, **meta: Any
```

Comment:
When we pass meta as an explicit dict we use `Dict[str, Any]` but when we pass it destructured as kwargs we use
`Any` , why not `**meta: Dict[str,Any]` ?

Reply:
`**foo` can't be typed like that, it would mean your local variable would be of type `Dict[str, Dict[str, Any]]`